data/events.xml

```diff
@@ -1,5 +1,59 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <events>
+    <event>
+        <lang>en</lang>
+        <startDate>2022-01-06</startDate>
+        <endDate>2022-01-06</endDate>
+        <location>San Diego, CA</location>
+        <speaker>Andy Goldberg</speaker>
+        <title>San Diego Kotlin User Group</title>
+        <subject>Kotlin APIs with Flexible Deployment Options</subject>
+        <url>https://www.meetup.com/sd-kotlin/events/mjfhnsydccbhb/</url>
+        <description>
+            <![CDATA[
+<p>Join us as Andy Goldberg talks about two packages, http4k and Hoplite, as used in combination. We'll walk through some code and some actual deployment options to see it in action. Anyone who would like a preview is welcome to read https://cfnine.com/blog/deploying-http4k.<p>
+]]>
+         </description>
+    </event>
+
+    <event>
```

Comment:
@TinaNEs it duplicates event from the previous commit https://github.com/JetBrains/kotlin-web-site/pull/2757/commits/55c2ac3b3c943c3949f3ca783d1813952580885a